packages/ui/src/Text/index.tsx

```diff
@@ -178,6 +203,19 @@ export type TextProps = TextType & {
   as?: WebTarget;
 };
 
+/**
+ * Runtime equivalent of TypeScript's `Omit` type. Removes extraneous props that
+ * shouldn't be passed to the DOM.
+ */
+const omit = <ObjectType extends Record<string, unknown>>(
+  object: ObjectType,
+  key: keyof ObjectType,
+) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars -- we're discarding the unused key
+  const { [key]: _, ...rest } = object;
+  return rest;
+};
+
```

Comment:
Created this to fix a previous issue where React was passing `h1`, `technical`, `large`, etc. as attributes to the DOM.